src/common/Requirements.ts

```diff
@@ -20,13 +20,15 @@ export interface Requirement {
     checkFunction: CheckRequirementsFunc;
     fulfilledMessage: string;
```

Comment:
Not too sure about it yet but we might be able to just remove `fulfilledMessage / unfulfilledMessage` since we're now going to print out the title + Passed/Failed

Reply:
Going to keep `fulfilledMessage / unfulfilledMessage`. But they are now optional allowing checks to not have messages at all.